test/test-helper/dummyFactories.js

```diff
@@ -39,9 +39,11 @@ Factory.define('User', User, {
 Factory.define('ObjSeq', DummyModel, {
   s1: Factory.seq(),
   s2: Factory.seq('ObjSeq.s2'),
-  s3: Factory.seq(),
+  s3: Factory.seq(n => n),
```

Comment:
Why is this required, isn't this default behaviour?

Reply:
@aexmachina you're right this would be the default behaviour. The idea is to test that passing a function as initializer would work as expected, so I'll change to something better 👍